VideoWeb/VideoWeb/Controllers/ParticipantsController.cs

```diff
@@ -137,14 +137,13 @@ public async Task<IActionResult> GetHeartbeatDataForParticipantAsync(Guid confer
         [SwaggerOperation(OperationId = "UpdateParticipantDisplayName")]
         [ProducesResponseType((int)HttpStatusCode.BadRequest)]
         [ProducesResponseType((int)HttpStatusCode.NoContent)]
-        public async Task<IActionResult> UpdateParticipantDisplayNameAsync(Guid conferenceId, Guid participantId,
-            [FromBody] UpdateParticipantDisplayNameRequest participantRequest)
+        public async Task<IActionResult> UpdateParticipantDisplayNameAsync(Guid conferenceId, Guid participantId, [FromBody] UpdateParticipantDisplayNameRequest participantRequest)
         {
             try
             {
-                var apiRequest = _mapperFactory.Get<UpdateParticipantDisplayNameRequest, UpdateParticipantRequest>()
-                    .Map(participantRequest);
-                await _videoApiClient.UpdateParticipantDetailsAsync(conferenceId, participantId, apiRequest);
+                var apiRequest = _mapperFactory.Get<UpdateParticipantDisplayNameRequest, UpdateParticipantRequest>().Map(participantRequest);
+                _videoApiClient.UpdateParticipantDetailsAsync(conferenceId, participantId, apiRequest).Wait(); //wait for update to complete before fetching participants to notify
```

Comment:
can we not use `await` here instead?

`await _videoApiClient.UpdateParticipantDetailsAsync(conferenceId, participantId, apiRequest);`

Reply:
I used wait, as the next statement fetches the updated conference data, and wanted to be sure this request completes before call ing the get request to retrieve the updated conference data

Reply:
Actually, I think you're right, shouldn't make a difference actually if it awaits the task completion before then next await. Was thinking about it in the wrong way

Reply:
but I think await should wait for the completion too. Just a bit conscious that `Wait()` could lead to deadlocks as it blocks the thread.

does await not work?

Reply:
sorry sent that last message at the last time as yours, disregard :)